app.js

```diff
@@ -12,21 +12,27 @@ const models = path.join(lib, 'models.js');
 
 const {Track} = require(models);
 
+const Configfile = require('electron-config');
+const configfile = new Configfile();
+
 app.on('ready', () => {
-  const folders = new Set([app.getPath('music')]);
   const acceptedExtensions = /\.mp3$/;
 
   const window = new BrowserWindow({ title: 'Bityn' });
   window.loadURL('file://' + templates + '/window-main.html');
   window.webContents.on('did-finish-load', function () {
-    const selectedFolders = dialog.showOpenDialog(window, {
-      title: 'Music folder',
-      defaultPath: app.getPath('music'),
-      properties: ['openDirectory']
-    });
-
-    if (selectedFolders !== undefined) {
-      folders.add.apply(folders, selectedFolders);
+    if (!configfile.has('folder')) {
+      var folders = dialog.showOpenDialog(window, {
```

Comment:
Don't use **var**, use **let** for mutable variables or **const** for non-mutable. I'll add this rule to the linter after this PR.